spec/models/mailing_spec.rb

```diff
@@ -1,26 +1,40 @@
 require 'spec_helper'
 
 describe Mailing do
-  let(:mailing) { Mailing.new(from: 'from@email.com', to: Role::ROLES,
+  let!(:mailing) { Mailing.new(from: 'from@email.com', to: Role::ROLES,
     cc: 'cc@email.com', bcc: 'bcc@email.com', subject: 'subject', body: '# body') }
 
   it { is_expected.to have_many(:submissions).dependent(:destroy) }
 
   describe '#sent?' do
-    skip 'TODO'
+
+    it 'false before submission' do
+      expect(mailing.subject).to eq 'subject'
+      mailing.save!
+      expect(subject.sent?).to eq false
+    end
+
+    it 'true after submitting' do
+      mailing.save!
+      mailing.submit
+      expect(mailing.sent_at).not_to be_nil
+      expect(mailing.submissions).not_to be_nil
+      expect(mailing.sent?).to eq true
+    end
   end
 
   describe '#submit' do
-    it 'should deliver seperate emails to all recipients (incl. cc and bcc)' do
+    it 'should deliver separate emails to all recipients (incl. cc and bcc)' do
       role = create :coach_role
       mailing.save!
       expect{mailing.submit}.to change{Submission.count}.by(mailing.emails.count)
```

Comment:
I don't understand what the `by(mailings.emails.count)` part is for... Why don't we use a fixed number here? I personally would have no idea what number and context this is talking about.